backend/src/main/java/gov/cdc/usds/simplereport/api/model/accountrequest/AccountRequest.java

```diff
@@ -133,28 +137,20 @@ public void setCellPhoneNumber(String cellPhoneNumber) {
     this.cellPhoneNumber = cellPhoneNumber;
   }
 
-  public String getMailingAddress1() {
-    return mailingAddress1;
+  public String getstreetAddress1() {
+    return streetAddress1;
```

Comment:
Would expect `public String getStreetAddress1()` for this and the setter.   However, instead, I suggest we use Jackson and remove all of these getters/setters (except `getEmail()` which is used elsewhere).